metadata.json

```diff
@@ -11,7 +11,7 @@
   "operatingsystem_support": [
     {
     "operatingsystem":"Windows",
-    "operatingsystemrelease":[ "2008R2", "2012", "2012R2" ]
+    "operatingsystemrelease":[ "2008R2", "2012", "2012R2", "7", "8" ]
```

Comment:
Probably could pull it out into it's own change. 7 and 8 should have powershell parity with the mentioned server versions. You really can't test that powershell is setup correctly, regardless of version, so it's really an all or none question in my mind, and I don't think none is reasonable. I did work on 7 and 8 to get this working and then used it on 2012.